### PR TITLE
Feat/no deploy dashboard with kdl

### PR DESCRIPTION
**Pull request by IGZangelcostales**

### CODE REVIEW

File: helm/science-toolkit/templates/common/rbac-config.yaml [hunk 1]
```diff
@@ -16,6 +16,7 @@ rules:
     resources: ["usertools"]
     verbs: ["*"]
 ---
+{{- if eq .Values.kdl.enabled "false" }}
```
Comment (danielchg) [inline helm/science-toolkit/templates/common/rbac-config.yaml:19]: .Values.kdl.enabled  should be treated as boolean, should be better something as follow
```
{{ if not .Values.kdl.enabled }}
```

File: helm/science-toolkit/templates/dashboard/dashboard-configmap.yaml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+{{- if eq .Values.kdl.enabled "false" }}
```
Comment (danielchg) [inline helm/science-toolkit/templates/dashboard/dashboard-configmap.yaml:1]: this configmap is used by kdl-server when deployed

File: helm/science-toolkit/templates/common/rbac-config.yaml [hunk 3]
```diff
@@ -75,4 +78,5 @@ subjects:
 roleRef:
   kind: Role
   name: dashboard
-  apiGroup: rbac.authorization.k8s.io
\ No newline at end of file
+  apiGroup: rbac.authorization.k8s.io
+{{- end }}
```
Comment (danielchg) [inline helm/science-toolkit/templates/common/rbac-config.yaml:82]: review las empty line

Comment (IGZangelcostales) [reply helm/science-toolkit/templates/common/rbac-config.yaml]: done

File: helm/science-toolkit/templates/dashboard/dashboard-configmap.yaml (hunk 2 above)
Comment (IGZangelcostales) [reply helm/science-toolkit/templates/dashboard/dashboard-configmap.yaml]: done

**Review by danielchg — APPROVED** [review]

**Pull request merged**